src/components/MessagesList/MessagesList.vue

```diff
@@ -1194,11 +1261,11 @@ export default {
 
 .messages-group {
 	&__date {
-		display: block;
-		text-align: center;
-		padding-top: 20px;
-		position: relative;
-		margin: 20px 0;
+		position: sticky;
+		top: 0;
+		display: flex;
+		justify-content: center;
+		z-index: 1000;
```

Comment:
`.top-bar {	z-index: 10; }`
`.dragover { z-index: 11; }`
Maybe, 9 would be enough?

Reply:
will see the minimum maxium z-index we can set comparing to other elements in the chat view